rllib/algorithms/algorithm.py

```diff
@@ -455,7 +455,7 @@ def __init__(
                 object. If unspecified, a default logger is created.
             **kwargs: Arguments passed to the Trainable base class.
         """
-        config = config or self.get_default_config()
+        config = config  # or self.get_default_config()
```

Comment:
nit: why remove the default config path?

Reply:
I see. Yes, this is impüortant when a user does not pass in any `config`. I will change this. Good catch!